.github/workflows/test-e2e-released.yml

```diff
@@ -0,0 +1,30 @@
+name: E2E (Released Kubeflex)
```

Comment:
It is best if there is a very close correspondence between the workflow name and the name of the file containing the workflow. For example, the proposed filename would work well if the workflow name were "Test E2E Released".